com.unity.cinemachine/Editor/Upgrader/UpgradeObjectToCm3.Data.cs

```diff
@@ -21,7 +21,15 @@ partial class UpgradeObjectToCm3
             typeof(CinemachineFreeLook),
             typeof(CinemachineVirtualCamera),
         };
-        
+
+        /// <summary>
+        /// Any component that may be referenced by vcams or freelooks 
+        /// </summary>
+        public static readonly List<Type> Referencables = new()
+        {
+            typeof(CinemachinePath)
```

Comment:
What about inemachineSmoothPath?

Reply:
Yes, it was meant to be the base class -> CinemachinePathBase.